app/api/activitylog/activitylogParser.ts

```diff
@@ -59,6 +59,10 @@ const entryValues: { [key: string]: EntryValue } = {
     idField: '_id',
     nameField: 'name',
   },
+  'POST/api/translations/import': {
+    desc: 'Uploaded translations csv',
```

Comment:
The message used in Entities import is "Imported entities from file", please could you use something like this to keep consistency.

Reply:
Changed to `Imported translations from file`